frontend/apps/crates/entry/user/src/register/pages/step_3/actions.rs

```diff
@@ -69,11 +69,9 @@ pub fn submit(state: Rc<State>) {
         affiliations,
         location: step_2
             .location_json
-            .map(|raw| serde_json::to_value(JsonRaw { raw }).unwrap_throw()),
+            .map(|raw| serde_json::to_value(raw).unwrap_throw()),
```

Comment:
Resolves the location data.

Reply:
Just making sure I understand correctly, this change isn't really related to the new trait, right?

Reply:
Yeah, this resolves the location not being shown in the users profile.